{
    "affected": [
        {
            "package": {
                "ecosystem": "Photon OS:2.0",
                "name": "python2",
                "purl": "pkg:rpm/vmware/python2?distro=photon-2"
            },
            "ranges": {
                "events": [
                    {
                        "introduced": "0"
                    },
                    {
                        "fixed": "2.7.15-1.ph2"
                    }
                ],
                "type": "ECOSYSTEM"
            }
        },
        {
            "package": {
                "ecosystem": "Photon OS:2.0",
                "name": "docker",
                "purl": "pkg:rpm/vmware/docker?distro=photon-2"
            },
            "ranges": {
                "events": [
                    {
                        "introduced": "0"
                    },
                    {
                        "fixed": "17.06.0-6.ph2"
                    }
                ],
                "type": "ECOSYSTEM"
            }
        },
        {
            "package": {
                "ecosystem": "Photon OS:2.0",
                "name": "strongswan",
                "purl": "pkg:rpm/vmware/strongswan?distro=photon-2"
            },
            "ranges": {
                "events": [
                    {
                        "introduced": "0"
                    },
                    {
                        "fixed": "5.5.2-5.ph2"
                    }
                ],
                "type": "ECOSYSTEM"
            }
        }
    ],
    "id": "PHSA-2018-0086",
    "modified": "2022-01-25T17:51:42Z",
    "published": "2018-08-24T00:00:00Z",
    "references": [
        {
            "type": "ADVISORY",
            "url": "https://github.com/vmware/photon/wiki/Security-Update-2.0-86"
        }
    ],
    "related": [
        "CVE-2018-10811",
        "CVE-2017-14992",
        "CVE-2018-1061",
        "CVE-2018-1060"
    ]
}